scripts/setup.sh

```diff
@@ -28,5 +39,5 @@ fi
 echo 'Updating .env.shared file (for shared configuration)...'
 aws s3 cp s3://artsy-citadel/dev/.env.metaphysics .env.shared || return
 
-echo 'Setup complete! To start the server on localhost:3000, run:
-  yarn dev'
+echo 'Setup complete! To start the server, run:
+  yarn start'
```

Comment:
`start` points at the `dev` command behind the scenes, so this keeps things a little more vanilla

Reply:
Vanilla is delicious.